### PR TITLE
[VEN-2843] Upgrade implementation of vToken contracts 

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -380,6 +380,16 @@ export const setMaxStalePeriod = async (
         normalTimelock,
         maxStalePeriodInSeconds,
       );
+
+      if (underlyingAsset.address === "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB") {
```
Comment (chechu) [inline src/utils.ts:384]: We could define a constant variable for this value, and use it here and in the file `src/vip-framework/checks/checkIsolatedPoolsComptrollers.ts`

File: src/utils.ts [hunk 2]
```diff
@@ -380,6 +380,16 @@ export const setMaxStalePeriod = async (
...
+        await setMaxStalePeriodInChainlinkOracle(
+          tokenConfig.oracles[1],
```
Comment (chechu) [inline src/utils.ts:386]: Why do we need to configure maxStalePeriod also for the pivot oracle?

File: multisig/simulations/ethereum/vip-063/index.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    vTokenBeacon = new ethers.Contract(VTOKEN_BEACON, VTOKEN_BEACON_ABI, ethers.provider);
+  });
+
+  describe("Pre-VIP behavior", () => {
+    it("check implementation", async () => {
+      expect(await vTokenBeacon.implementation()).to.be.equal("0xE5A008B6A0bAB405343B3ABe8895966EAaFb5790");
+    });
+  });
+
+  describe("Post-VIP behavior", async () => {
```
Comment (GitGuru7) [inline multisig/simulations/ethereum/vip-063/index.ts:23]: Can we also check based on time and the number of blocks for additional validation purpose?

File: src/utils.ts (hunk 2 above)
Comment (kkirka) [reply src/utils.ts:386]: Just to confirm, `checkIsolatedPoolsComptroller` fails even without this VIP, right? Do you know where we have underlying asset == BNB (and not WBNB) in IL?

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/378?chainId=56

**Review by chechu — APPROVED** [review]

**Pull request merged**